fe/src/app/pages/homepage/homepage.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="hero-banner">
+  <img
+    src="images/homepage-banner.jpg"
+    alt="Foto von einem Alpaka"
+    class="hero-image"
+    aria-hidden="true" />
+
+  <div class="hero-overlay"></div>
+
+  <div class="hero-text">
+    <h1>Willkommen auf dem Rosenhof</h1>
+    <p>Erlebe unsere Alpakas hautnah!</p>
+  </div>
+</div>
+
+<div class="content-container homepage-sections">
+  <section class="feature-section">
+    <div class="feature-image">
+      <img
+        src="images/alpakas.jpg"
+        alt="Zwei flauschige Alpakas schauen freundlich in die Kamera" />
+    </div>
+    <div class="feature-text">
+      <h2>Willkommen bei den Alpakas vom Rosenhof</h2>
+      <p>Wir freuen uns, mit euch ein Alpaka-Abenteuer zu erleben.</p>
+      <p>
+        Unser Hof befindet sich im Lampertheimer Ortsteil "Rosengarten" nahe der Wormser
+        Rheinbrücke.
+      </p>
+      <p>
+        Wenn ihr Interesse daran habt, <a routerLink="/animals">unsere Tiere</a> live
+        kennenzulernen, findet ihr <a routerLink="/about">hier</a> unsere aktuellen Events und
+        Termine.
+      </p>
+      <p>
+        Schaut doch mal auf unserer Instagram-Seite vorbei, hier posten wir regelmäßige Updates und
+        Bilder rund um unsere Tiere.
+      </p>
+      <a
+        href="https://www.instagram.com/alpakas_vom_rosenhof"
+        target="_blank"
+        rel="noopener noreferrer"
+        class="feature-button">
+        <fa-icon [icon]="icons.faInstagram"></fa-icon>
+        <span>Instagram</span>
+      </a>
+    </div>
+  </section>
+
+  <section class="feature-section feature-section--reversed">
+    <div class="feature-text">
```

Comment:
Eventuell können wir das später mal in eine eigene Component packen und im Backend konfigurieren welche Features auf der Startseite angezeigt werden